tests/unit/core/statistics/datatypes/test_floattracker.py

```diff
@@ -43,3 +67,55 @@ def test_merge_floattrackers_should_addup():
 
     merge_second = second.merge(first)
     assert merge_second.__dict__ == merge_first.__dict__
+
+def test_merge_floattrackers_should_addup():
+    float32_tracker = FloatTracker()
+    float32_array = np.array([1.0, 2.0, 3.0], dtype=np.float32)
+    for val in float32_array:
+        float32_tracker.update(val)
+
+    float64_tracker = FloatTracker()
+    float64_array = np.array([1.0, 2.0, 3.0], dtype=np.float64)
+    for val in float64_array:
+        float64_tracker.update(val)
+
+    simple_tracker = FloatTracker()
+    vals2 = [4.0, 5.0, 6.0]
+    for val in vals2:
+        simple_tracker.update(val)
+
+    assert simple_tracker.count == len(vals2)
```

Comment:
aren't these assertions already covered up on `test_values_are_min_max()`?

Reply:
I was thinking the same thing :) I was curious why the first test did it that way. I'll update as I was feeling weird about that as well. 

What I would like to do for the testing is break this up so that we can send in the type and test like and not have to have so much duplicating.